### DISCUSSION TITLE
Why wasn't a 'import old .data files' feature included in 3.0.0? Why doesn't v3.0.0 automatically import old .data files gracefully on first run?

### DISCUSSION THREAD

**Discussion by cinardoruk** [GothenburgBitFactory/taskwarrior#3358] [General]
I do a regular system update (with 100+ packages, as usual. who has the time to read release notes for every single package before doing an update?) and now all my tasks are gone. Oh wait, they are still there in $HOME/.task but just aren't recognized by taskwarrior.
I read the notes and ooooooooooooooh: 'you should export your tasks before upgrading to v3.0.0 and then re-import them'
Thanks for wasting hours of so many people's time and causing incomprehensible amounts of frustration with extra programmer rage sprinkled on top. Great job.

**Comment by djmitche** [comment]
This comment reflects some understandable frustration, and I'm sorry for that. I have felt similarly with other projects and tools.
